proxies/shared/resources/jsc/helpers/validationChecks.js

```diff
@@ -106,21 +106,6 @@ const validateArray = (errors, fieldValue, fieldPointer, minElements) => {
     return true
 }
 
-const validateDob = (errors, fieldValue, fieldPointer) => {
-    if (
-        !isUndefined(fieldValue)
-        && (
-            fieldValue === null
-            || typeof fieldValue !== "string"
-            || !dobRegex.test(fieldValue)
```

Comment:
Looks like you can also remove line 4 in this file?
```
const dobRegex = /^\d{4}-\d{2}-\d{2}$/i
```

Reply:
Nice catch! Done.